client/src/includes/panels.ts

```diff
@@ -43,16 +44,23 @@ const toggleCollapsiblePanel = (
  * Initialises event handlers for a collapsible panel,
  * and applies the correct initial state based on classes.
  */
-function initCollapsiblePanel(toggle: HTMLButtonElement) {
+export function initCollapsiblePanel(toggle: HTMLButtonElement) {
   const panel = toggle.closest<HTMLElement>('[data-panel]');
   const content = document.querySelector<HTMLDivElement>(
     `#${toggle.getAttribute('aria-controls')}`,
   );
 
-  if (!content || !panel) {
+  // Avoid initialising the same panel twice.
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  if (!content || !panel || panel.collapsibleInitialised) {
```

Comment:
Not my proudest work but this helps resolve double initialisation issues with this component. This is exactly the type of thing that won’t be an issue anymore once we switch over to Stimulus, which would be a great fit for this component, so I did the quick fix and moved on.